usage.md

```diff
@@ -16,9 +16,9 @@ Installation will be required for capabilities such as [self-provisioning](doc/d
 
 Replace the `<fields>` in the example with values specific to your environment - this will install VCH to the specified resource pool of ESXi or vCenter, and the container VMs will be created under that resource pool. Bug #539 is tracking the container VM creation issue.
 
-Here is one resource pool path sample: /ha-datacenter/host/localhost/Resources/test 
--generate-cert flag is to generate certificates and configure TLS
--force flag is to remove an existing datastore folder or VM with the same name:
+- Here is one resource pool path sample: /ha-datacenter/host/localhost/Resources/test.
```

Comment:
suggest you add \`'s around the path rather than put a . at the end - it makes it easier to differentiate between value and simple punctuation.